safetrace-sdk/Services/Session/UserSession.swift

```diff
@@ -73,13 +73,14 @@ class UserSession: UserSessionProtocol {
     private let groupKeychain: KeychainProtocol
     
     init(
-        environment: Environment,
+        environment: Environment & UserSessionAuthenticationDelegate,
```

Comment:
@maxmamis Was thinking about this some more, maybe it made more sense for the `authenticationDelegate` to be set from outside of this initializer, in the `TracerEnvironment`